RecoLocalCalo/CaloTowersCreator/src/CaloTowersCreator.cc

```diff
@@ -470,6 +487,6 @@ void CaloTowersCreator::fillDescriptions(edm::ConfigurationDescriptions& descrip
   desc.add<std::vector<std::string> >("EcalSeveritiesToBeUsedInBadTowers", {});
   desc.add<int>("HcalPhase", 0);
   desc.add<bool>("usePFThresholdsFromDB", true);
-
+  desc.add<bool>("EcalRecHitThresh", true);
   descriptions.addDefault(desc);
```

Comment:
Unrelated, but why is this```addDefault```  not ```addWithDefaultLabel``` or similar?

Reply:
I can't say why it's the case, but it's part of the more general https://github.com/cms-sw/cmssw/pull/44372/ (to which I still plan to come back, when time allows).